### ISSUE TITLE
Fix windows php 5.6 build

### ISSUE THREAD

**Issue by phadej** [igbinary/igbinary#37]

**Comment by Rewt0r** [comment]
http://windows.php.net/downloads/pecl/releases/igbinary/1.2.1/ no build for php 5.6

**Comment by slifin** [comment]
are we still waiting for a 5.6 build?

**Comment by TysonAndre** [comment]
This is related to https://github.com/igbinary/igbinary/issues/44

APC PECL doesn't support PHP 5.6, so the version that is checked out has compiler errors for undefined constants. (Replaced with IS_CONSTANT_AST, other constants).
(Last APC release was in 2012: https://pecl.php.net/package/APC)

PHP 5.6 uses opcache instead for opcode caching.

Switching to APCU for PHP 5.6, 7.x is one option. (Keeps apcu_fetch/store,etc.)

I don't have a windows environment set up for testing changes.

```
ext\apc\apc_cache.c(215) : warning C4244: 'initializing' : conversion from 'time_t' to 'int', possible loss of data
ext\apc\apc_compile.c(253) : error C2065: 'IS_CONSTANT_INDEX' : undeclared identifier
ext\apc\apc_compile.c(254) : warning C4267: '=' : conversion from 'size_t' to 'int', possible loss of data
ext\apc\apc_compile.c(292) : warning C4267: 'function' : conversion from 'size_t' to 'int', possible loss of data
ext\apc\apc_compile.c(356) : error C2065: 'IS_CONSTANT_ARRAY' : undeclared identifier
ext\apc\apc_compile.c(356) : error C2051: case expression not constant
ext\apc\apc_cache.c(818) : error C2065: 'IS_CONSTANT_INDEX' : undeclared identifier
ext\apc\apc_cache.c(820) : error C2065: 'IS_CONSTANT_ARRAY' : undeclared identifier
ext\apc\apc_cache.c(820) : error C2051: case expression not constant
```

**Comment by TysonAndre** [comment]
For some reason, http://windows.php.net/downloads/pecl/releases/igbinary/2.0.0/logs/ only shows logs for trying to build for php 7.0, and not for php 5.x

I'm not sure if building for 7.0 is the new default, or if pecl stopped trying because the 7.0 builds failed, or if it's because of a dependency on APCU (and apcu 5.x requiring >= 7.0. I don't see a declared dependency on apcu, which is odd)
I didn't see any options for package.xml 2.0 that would help in the pecl documentation.

https://github.com/igbinary/igbinary/pull/108 may fix the 7.0 builds in the next release.

**Comment by TysonAndre** [comment]
https://pecl.php.net/package/igbinary/2.0.1/windows has DLLs for 5.5, 5.6, 7.0, and 7.1 right now.
